website/docs/reference/resource-configs/database.md

```diff
@@ -45,6 +45,38 @@ This would result in the generated relation being located in the `staging` datab
 </File>
 </TabItem>
 
+<TabItem value="snapshots" label="Snapshots">
+
+<VersionBlock lastVersion="1.8">
+
+Available for versionless dbt Cloud or dbt Core v1.9+. Select v1.9 or newer from the version dropdown to view the configs.
```

Comment:
Confirming are we calling this "versionless" or "Keep on latest version"? cc: @jtcohen6

Reply:
Good question - we went back earlier this quarter and added "versionless" in front of every mention of "Keep on latest version" in the docs to tie the two ideas together but I'll defer to Jerco about how to handle the naming going forward

Reply:
Oh if that's already added, then that works for me!